src/Client/Features/OfferMatching/List.razor.cs

```diff
@@ -30,7 +32,26 @@ protected override async Task OnInitializedAsync()
             _loading = false;
         }
     }
+    private async Task LoadDataAsync()
+    {
+        _result = await HttpClient.GetFromJsonAsync<OfferMatchingListResult>(OfferMatchingListRouteFactory.Uri, _cts.Token) ?? new();
+    }
+
+    private async void MatchPlayerAsync(int playerId, int amount)
+    {
+        _player = new MatchPlayerRequest() { PlayerId = playerId, Amount = amount };
```

Comment:
not sure i follow why this is a private variable and the assignment versus a local variable.